extensions-ocp-rhel-9.6.yaml

```diff
@@ -20,9 +20,7 @@ extensions:
   # https://github.com/coreos/fedora-coreos-tracker/issues/1504
   ipsec:
     packages:
-      # pin to 4.6-3.el9_0.3 for now for https://issues.redhat.com/browse/OCPBUGS-43498
-      # we can revert once that's fixed in latest libreswan
-      - libreswan-4.6-3.el9_0.3
+      - libreswan-5.2-1.el9fdp
```

Comment:
Instead of binding the version this way, we should just remove the pin here, i.e. remove the version.  Once libreswan 5 is available in FDP, it will be automatically installed and we'll consume all the bug fixes automatically once they are available.  Pinning a specific version is not a good long term solution.

Reply:
@igsilya sure, will do, please let us know once we have libreswan 5 is available in FDP, then will remove pinning to specific version. cc @The-Mule